builtin/logical/transit/path_rotate.go

```diff
@@ -56,9 +56,12 @@ func (b *backend) pathRotateWrite(ctx context.Context, req *logical.Request, d *
 		Name:    name,
 	}, b.GetRandomReader())
 	if err != nil {
+		// the error here will be something about "couldn't get policy")
+		b.Logger().Error("failed to rotate key on user request", "name", name, "err", err.Error())
```

Comment:
Should we use `error` for all these so that all new log entries are consistent in Vault?

Reply:
yep, changed